resolwe_bio/processes/support_processors/insert_size.py

```diff
@@ -26,7 +26,7 @@ class InsertSizeMetrics(Process):
     name = 'Picard InsertSizeMetrics'
     category = 'Picard'
     process_type = 'data:picard:insert'
-    version = '1.0.0'
+    version = '1.0.1'
```

Comment:
The same as in previous commit. Bump the workflow version, not the process version.